integration-libs/opf/checkout/components/opf-checkout-payment-and-review/opf-checkout-payment-and-review.component.ts

```diff
@@ -87,17 +71,6 @@ export class OpfCheckoutPaymentAndReviewComponent
       .pipe(map((cart: Cart) => cart.paymentType));
   }
 
-  constructor() {
-    super(
-      this.checkoutDeliveryAddressFacade,
-      this.checkoutPaymentFacade,
-      this.activeCartFacade,
-      this.translationService,
-      this.checkoutStepService,
-      this.checkoutDeliveryModesFacade
-    );
-  }
-
```

Comment:
We can reuse implicit constructor of the parent class, by skipping a constructor definition in the child class. So no need to re-declare all deps for the parent class. All `protected` properties are inherited anyway.